modules/load-balancer-target-group/variables.tf

```diff
@@ -50,10 +50,10 @@ variable "vpc_id" {
 
 variable "routing_condition" {
   description = "This variable defines the paths or domain names that will be routed to the servers. By default, we route all paths and domain names to the servers. To override this, you should pass in a list of maps, where each map has the keys field and values. See the Condition Blocks documentation for the syntax to use: https://www.terraform.io/docs/providers/aws/r/lb_listener_rule.html."
-  type = list(object({
+  type = object({
     field  = string
     values = list(string)
-  }))
+  })
```

Comment:
Ah sorry about that... I misread the default 🤦‍♂

Reply:
Yeah its fine 😀